frontend/src/scenes/surveys/SurveyCustomization.tsx

```diff
@@ -134,6 +135,43 @@ export function Customization({ appearance, surveyQuestionItem, onAppearanceChan
                         checked={appearance?.shuffleQuestions}
                     />
                 </div>
+                <div className="mt-1">
+                    <LemonField name="survey_popup_delay" className="font-medium">
+                        {({ onChange }) => {
+                            return (
+                                <div className="flex flex-row gap-2 items-center">
+                                    <LemonCheckbox
+                                        checked={!!appearance?.surveyPopupDelay}
+                                        onChange={(checked) => {
+                                            const surveyPopupDelay = checked ? 5 : undefined
+                                            onChange(surveyPopupDelay)
+                                            onAppearanceChange({ ...appearance, surveyPopupDelay })
+                                        }}
+                                    />
+                                    Delay survey popup after page load by{' '}
+                                    <LemonInput
+                                        type="number"
+                                        data-attr="survey-popup-delay-input"
+                                        size="small"
+                                        min={1}
```

Comment:
add a sensible max as well? If this is seconds, I think anything above 1 hour is crazy :P